src/contexts/index.ts

```diff
@@ -25,5 +25,6 @@ export const RefreshControlComponentContext = React.createContext<
 >(undefined);
 
 export const DocContext = React.createContext<{
-  getDoc: () => TypesLegacy.Document;
+  getDoc: () => TypesLegacy.Document | undefined;
```

Comment:
FYI - this signature was somehow incorrect - you can see the context provider was passing a function instead of an object with `getDoc` key. We need to update the other places in the code where the context consumer calls that function directly (`this.context()`)